package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "pcd-snark-artifacts-root",
+  "version": "0.0.0",
+  "repository": "https://github.com/proofcarryingdata/snark-artifacts.git",
+  "license": "MIT",
+  "private": true,
```

Comment:
not sure if this causes problems but suggesting because it makes sense to me

Reply:
I think what this does is make sure NPM will refuse to publish this "package", which isn't something we want for the root of the repo.